lib/msf/core/opt_bool.rb

```diff
@@ -8,7 +8,7 @@ class OptBool < OptBase
 
     # This overrides default from 'nil' to 'false'
     def initialize(in_name, attrs = [],
-                   required: true, desc: nil, default: false, aliases: [])
+                   required: true, desc: nil, default: false, conditions: [], aliases: [])
```

Comment:
Since these don't need to care about the particular keywords aside from the ones they touch, we can take this opportunity to wildcard them to make the method less fragile for the future:
```suggestion
                   default: false, **kwargs)
```